wiki/Ranking_Criteria/osu!/en.md

```diff
@@ -36,7 +36,10 @@ Rule proposals as well as suggestions are discussed in the [Ranking Criteria Sub
 - **Spinner recovery time:** The time between the end of a spinner and the following hit object.
 - **Time-distance equality:** Spacing between hit objects is proportional to the duration of time between them. Also known as using a constant distance spacing.
 - **Hit object/Note density:** The average timeline gaps between hit objects expressed in fractions of a beat (such as 1/1 for one beat, 1/2 for half a beat).
-- **Key sounding:** hit sounding using samples extremely similar or directly from the song in their respective places. This method of hit sounding usually provides low feedback to the player.
+- **Key sounding:** Hit sounding using samples extremely similar or directly from the song in their respective places. This method of hit sounding usually provides low feedback to the player.
+- **Spaced stream:** A stream whose circles do not overlap.
```

Comment:
```suggestion
- **Spaced stream:** A stream with circles that do not overlap.
```
[current version should be fine](https://www.merriam-webster.com/words-at-play/whose-used-for-inanimate-objects), but it implies that none of the circles overlap; idk if that was the intention. are accelerating streams considered "spaced streams" if there are some overlaps at the beginning?